evalml/tests/component_tests/test_utils.py

```diff
@@ -31,26 +31,26 @@ def test_all_components(
     is_using_conda,
     is_using_windows,
 ):
-    # The total number of minimal components is 41
-    # The total number of components is 54
+    # The total number of minimal components is 43
+    # The total number of components is 56
     # Depending on the environment the detrender/Arima and/or Prophet will not be installed
 
     if has_minimal_dependencies:
-        n_components = 41
+        n_components = 43
     elif is_using_conda:
         # No prophet and no arima
-        n_components = 52
+        n_components = 54
     elif is_using_windows and not is_running_py_39_or_above:
         # No prophet
-        n_components = 53
+        n_components = 55
     elif is_using_windows and is_running_py_39_or_above:
         # No detrender, no arima, no prophet
-        n_components = 51
+        n_components = 53
```

Comment:
Since these are Windows/Conda tests, we might just not get coverage for this section until we merge to main.